CITATION.cff

```diff
@@ -0,0 +1,25 @@
+cff-version: 1.2.0
+message: "Please cite the following works when using this software."
+type: software
+title: "hist: v2.6.1"
+abstract: "Hist is an analyst-friendly front-end for boost-histogram, designed for Python 3.7+."
+authors:
+- family-names: "Schreiner"
+  given-names: "Henry"
+  orcid: "https://orcid.org/0000-0002-7833-783X"
+  affiliation: "Princeton University"
+- family-names: "Liu"
+  given-names: "Shuo"
+  affiliation: "Columbia University in the City of New York"
```

Comment:
@LovelyBuggies can you please confirm that this information is correct? Also, do you have an [ORCID](https://orcid.org/)?